components/build-service/staging/base/kustomization.yaml

```diff
@@ -3,3 +3,14 @@ kind: Kustomization
 resources:
 - ../../base
 - ../../base/external-secrets
+- https://github.com/redhat-appstudio/build-service/config/default?ref=45e8fe7c3b036d1df3eee11c1d83e0633e451c07
+
+namespace: build-service
+
+images:
+- name: quay.io/redhat-appstudio/build-service
+  newName: quay.io/redhat-appstudio/build-service
+  newTag: 45e8fe7c3b036d1df3eee11c1d83e0633e451c07
+
+commonAnnotations:
+  argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
```

Comment:
missing new line

Reply:
the pr got merged now.. will address this in a future PR